cmake/scripts/linux/platform/X11.cmake

```diff
@@ -1,8 +1,3 @@
-core_require_dep(OpenGl)
-core_require_dep(EGL)
-core_require_dep(X)
-core_require_dep(XRandR)
-core_require_dep(LibDRM)
-core_optional_dep(VAAPI)
-core_optional_dep(VDPAU)
-core_optional_dep(OpenGLES)
+set(ENABLE_X11 ON)
```

Comment:
wouldn't it be better to change the relevant places to use the variable X_FOUND?